### ISSUE TITLE
patches description

### ISSUE THREAD

**Issue by scippio** [wunderkraut/alpine-nginx-pagespeed#17]
Hi all, I know the patches is small and maybe self explanatory, but I don't fully understand to build process ... so it should be nice have some small description about patches. thx... I love this docker image :)

**Comment by aleksijohansson** [comment]
Good idea, I'll try to do this when the more pressing issues are solved.
